static/js/protein/data_processing_utils.ts

```diff
@@ -192,6 +192,9 @@ export function getDiseaseGeneAssoc(data: GraphNodes): ProteinNumData[] {
         for (const n1 of n.nodes) {
           for (const n2 of n1.neighbors) {
             if (n2.property === "diseaseOntologyID") {
+              if (_.isEmpty(n2.nodes)) {
```

Comment:
I don't think this is necessary because if n2.nodes is empty, the code won't go through the next for loop anyways

Reply:
But wouldn't we want to check that n2.nodes is non-empty before we run the next for loop with n3 ?

Reply:
the next for loop looks at each node in n2.nodes, but if there's no nodes, it just won't do anything because there's no nodes to look at?

Reply:
I see ! You're right, removed the unnecessary check !